flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/rest/serde/LogicalTypeJsonSerDeTest.java

```diff
@@ -219,6 +231,25 @@ private static List<LogicalType> generateTestData() {
         // ignore nullable for NullType
         testTypes.add(new NullType());
 
+        // interval types
+        testTypes.add(new YearMonthIntervalType(YearMonthIntervalType.YearMonthResolution.MONTH));
```

Comment:
We should move the cases into the `types`. In line 225, we will create test cases with or without nullbility.

Reply:
done